debug.md

```diff
@@ -14,13 +14,15 @@ The default remote debug port is 40000 but its configurable by setting the `DEBU
 export DEBUG_PORT=8765
 ```
 
-Compile the code with debug information, patch the image and push to your local k8s:
+Build the image and push to your local k8s:
```

Comment:
I think that this doc belongs under https://github.com/openshift/assisted-service/tree/master/docs/dev